.github/workflows/merge-queue.yml

```
@@ -3,8 +3,100 @@ name: Validate code in the merge queue (post merge)
+      - opened
+defaults:
+  run:
+    shell: bash -euxo pipefail {0}
 
 jobs:
+  prepare:
+    runs-on: ec2-runners
+    container:
+      image: public.ecr.aws/m3u4c4h9/island-is/actions-runner-public:latest
+    outputs:
+      TEST_CHUNKS: ${{ steps.get_cache.outputs.CHUNKS }}
```

actionlint (1.7.4)

property "chunks" is not defined in object type {keys: string}

(expression)